app/services/file_mover.py

```
@@ -401,7 +405,9 @@ def move_with_rollback(
             symlink_target = translate_path_for_symlink(str(destination))
             source.symlink_to(symlink_target)
         except OSError as e:
-            logger.error("Failed to create symlink at %s pointing to %s: %s", source, destination, e)
+            logger.error(
+                "Failed to create symlink at %s pointing to %s: %s", source, destination, e
+            )
```

Ruff (0.15.14)

[warning] Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)